priv/repo/migrations/20220505114827_automation_setup_scope.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Oli.Repo.Migrations.AutomationSetupScope do
+  use Ecto.Migration
+
+  def change do
+    alter table(:api_keys) do
+      add :automation_setup_enabled, :boolean, default: false, null: false
+    end
+
+    flush()
+
+    execute "UPDATE api_keys SET automation_setup_enabled = false;"
```

Comment:
Isn't the `default: false` above doing the same as this line?

Reply:
I was copying what Oli.Repo.Migrations.RegistrationScope did and assumed it had some purpose. I'll try without to see.